### DISCUSSION TITLE
How do I disable how it shows fps and other diagnostic information

### DISCUSSION THREAD

**Discussion by Raj2032** [SuperTux/supertux#2265] [Q&A] [answered]
For some odd reason on Windows, it shows fps and other diagnostic information on the top right hand side, any ideas how to disable all of this?

**Comment by provigz** [comment, marked as answer]
Make sure Developer Mode is enabled, by pressing F2 in-game. If no menu appears, execute the key combination Ctrl+F2 to enable Developer Mode, and try again. You should see a debug options menu. You can toggle the FPS counter visibility and other diagnostic information from there.
